### ISSUE TITLE
Docs are missing information about how one authorizes

### ISSUE THREAD

**Issue by TimonLukas** [Watson-Personal-Assistant/developer#10]
The docs are currently missing information about how you authorize yourself if you try to make an API request against Watson Assistant. This is especially confusing since the Swagger examples to not have the API key when you execute them, and the error talks about an authorization, even though auth happens using a GET parameter.

Do change this I'd suggest that a new paragraph is added, e. g. to the converse page, which explains how you can authorize external requests to the service.

**Comment by troyibm** [comment]
Yes, we have an issue open (probably iceboxed by now or close) where the curl command on the swagger doesn't have the api_key header or added as a query parameter to the URL so copy and paste the curl commands don't work.

That said, we are moving away from providing API keys and instead requiring IBMid to log into the service.  And, I think, that will cause the curl commands not to work at all.  I'll have to investigate the plan for use curl commands without API keys.  Our curl days might be numbered.
